### PR TITLE
constrain IMU filter length

### PR DESCRIPTION
**Pull request by zakmat**
When the IMU's filter length is set to 0 sensor fails to publish data. This PR is adding a constraint on the IMU component's `Filter Length` property as only positive values make sense

### CODE REVIEW

File: Gems/ROS2/Code/Source/Imu/ROS2ImuSensorComponent.cpp [hunk 1]
```diff
@@ -50,6 +50,7 @@ namespace ROS2
                         &ROS2ImuSensorComponent::m_filterSize,
                         "Filter Length",
                         "Filter Length, large value allows to reduce numeric noise")
+                    ->Attribute(AZ::Edit::Attributes::Min, 1)
```
Comment (spham-amzn) [inline Gems/ROS2/Code/Source/Imu/ROS2ImuSensorComponent.cpp:53]: Is there also a max value we can attribute here as well?

Comment (adamdbrw) [reply Gems/ROS2/Code/Source/Imu/ROS2ImuSensorComponent.cpp:53]: I guess it does not make much sense to have a very large moving window. Maybe 200 could be a good maximum?

Comment (michalpelka) [reply Gems/ROS2/Code/Source/Imu/ROS2ImuSensorComponent.cpp:53]: Agree, add 200 maximum samples it will be 2 seconds for 100 Hz.

Comment (zakmat) [reply Gems/ROS2/Code/Source/Imu/ROS2ImuSensorComponent.cpp:53]: I have added a max value attribute at 200 and changed UIHandler to Slider for a better user experience

**Review by adamdbrw — APPROVED** [review]

**Review by michalpelka — APPROVED** [review]

**Pull request merged**